conflux-sponsor-faucet-sdk/faucet-frontend/faucet.js

```diff
@@ -105,24 +105,25 @@ class Faucet {
      * @dev get bounds and limit params of faucet
      */
     async getFaucetParams() {
+        let e = await this.cfx.getEpochNumber('latest_confirmed');
```

Comment:
By default, the 2nd parameter of `cfx_call` is unnecessary, which means the latest state.

Note, data of `latest_confirmed` may be out of date.

Reply:
rm params